### PR TITLE
Add a test for content-dpr when served for a tiled-background image

### PR DESCRIPTION
**Pull request by noamr**
@yoavweiss

### CODE REVIEW

File: content-dpr/tiled-background-image-with-content-dpr.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    <meta name="assert" content="Assert that content-dpr is taken onto account for tiled background images">
+    <style>
+      #bg, #bg2 {
+        width: 32px;
+        height: 32px;
+        background-image: url(resources/dpr.py?name=background.png&mimeType=image/png&dpr=2.0);
+      }
+
+      #bg2 {
+        background-size: 8px 8px;
```
Comment (yoavweiss) [inline content-dpr/tiled-background-image-with-content-dpr.html]: Did you mean to also include the image here?

File: content-dpr/tiled-background-image-with-content-dpr-ref.html [hunk 2]
```diff
@@ -0,0 +1,30 @@
+<html>
+  <head>
+    <title>Content-DPR: css tiled background</title>
+    <link rel="author" title="Noam Rosenthal" href="noam@webkit.org">
+    <link rel="match" href="tiled-background-image-with-content-dpr-ref.html" />
+    <meta name="assert" content="Assert that content-dpr is taken onto account for tiled background images">
```
Comment (yoavweiss) [inline content-dpr/tiled-background-image-with-content-dpr-ref.html]: Nit: s/onto/into/

File: content-dpr/tiled-background-image-with-content-dpr.html (hunk 1 above)
Comment (noamr) [reply content-dpr/tiled-background-image-with-content-dpr.html]: It's there in the previous CSS declaration

File: content-dpr/tiled-background-image-with-content-dpr-ref.html (hunk 2 above)
Comment (noamr) [reply content-dpr/tiled-background-image-with-content-dpr-ref.html]: gottit

Comment (noamr) [reply content-dpr/tiled-background-image-with-content-dpr-ref.html]: fixed

File: content-dpr/tiled-background-image-with-content-dpr.html (hunk 1 above)
Comment (noamr) [reply content-dpr/tiled-background-image-with-content-dpr.html]: (same image, just with/without background-size)

Comment (yoavweiss) [reply content-dpr/tiled-background-image-with-content-dpr.html]: you're right, of course! Missed it earlier

**Review by yoavweiss — APPROVED** [review]
LGTM

**Pull request merged**